code/modules/admin/admin_verbs.dm

```diff
@@ -619,6 +623,58 @@ var/list/admin_verbs_cciaa = list(
 	message_admins("<span class='notice'>[ckey] creating an admin explosion at [epicenter.loc].</span>")
 	feedback_add_details("admin_verb","DB") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cure_traumas(mob/T as mob in mob_list)
+	set category = "Fun"
+	set name = "Cure Traumas"
+	set desc = "Cure the retardations of a given mob."
+
+	if(!istype(T,/mob/living/carbon/human))
+		to_chat(usr, "This can only be done to instances of type /mob/living/carbon/human")
+		return
+
+	var/mob/living/carbon/human/C = T
+
+	C.cure_all_traumas(TRUE, TRUE)
+	var/ceekeey = "NO CKEY"
+	if(C.ckey)
+		ceekeey = C.ckey
+	message_admins("<span class='notice'>[ckey] cured [C]'s ([ceekeey]) traumas.</span>")

```

Comment:
This isn't logging to the server log, I'd change it to:
```dm
log_and_message_admins("<span class='notice'>cured [key_name(C)]'s traumas.</span>")
```
(The proc will prefix the admin's ckey automatically).